ripples/multi_session.py

```
@@ -25,403 +32,1150 @@
+        for cell_type in cell_types:
+            ind_area = [
+                get_clusters_ind_per_region(session, area) for session in all_genotypes
+            ]
+            spike_times_per_session = [
+                [
+                    np.array(cluster.spike_times)
+                    for cluster in all_genotypes[session_idx].clusters_info[
+                        min(ind_area[session_idx]) : max(ind_area[session_idx])
+                    ]
+                    if np.logical_or(cluster.cell_type == cell_type, cell_type == "A")
+                    if cluster.good_cluster == True
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if cluster.good_cluster:` for truth checks

Replace with `cluster.good_cluster`

(E712)